roles/monitoring/templates/user-values-elasticsearch-open.yaml

```diff
@@ -25,7 +28,10 @@ elasticsearch:
     ingress:
       annotations:
         nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+        kubernetes.io/ingress.class: nginx
       enabled: true
+      # ingressClassName is not currently supported by the Open Distro helm chart
+      # ingressClassName: nginx
```

Comment:
Are these needed for the future or place holders?

Reply:
Yes, these were place holders in the sample files provided by the viya4-monitoring-kubernetes, I kept them same in our files.